Jenkinsfile

```
@@ -30,6 +30,22 @@ boolean doc_only_change() {
     return rc == 1
 }
 
+boolean release_candidate() {
+    return !sh(label: "Determine if building (a PR of) an RC",
+              script: "git diff-index --name-only HEAD^ | grep -q TAG && " +
+                      "grep -i '[0-9]rc[0-9]' TAG",
+              returnStatus: true)
+}
+
+def scons_faults_args() {
+    // The default build will have BUILD_TYPE=dev; fault injection enabled
+    if ((cachedCommitPragma(pragma: 'faults-enabled', def_val: 'true') == 'true') && !release_candidate()) {
```

Oh, pity that I missed this:

```suggestion
    if ((cachedCommitPragma(pragma: 'Faults-enabled', def_val: 'true') == 'true') && !release_candidate()) {
```
as that would have followed the format of all previous pragrmas with a capital initial letter.

@mjean308 Can you please create a PR for this, or include that change in some soon upcoming PR?  And update the Wiki?  Please and thanks.